trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/validate-facility-id-query.ts

```diff
@@ -2,26 +2,28 @@ import { isNonEmptyString } from '@ukef/dtfs2-common';
 import { ErrorSummaryViewModel } from '../../../types/view-models';
 import { REGEX } from '../../../constants';
 
-const FACILITY_ID_INPUT_ID = '#facility-id-filter';
-
-export const validateFacilityIdQuery = (facilityIdQuery: string | undefined, originalUrl: string): ErrorSummaryViewModel | undefined => {
-  if (originalUrl.includes('?') && originalUrl.includes('facilityIdQuery')) {
+export const validateFacilityIdQuery = (
+  facilityIdQuery: string | undefined,
```

Comment:
```suggestion
  facilityIdQuery?: string,
```

Reply:
Thanks, updated this on another WIP branch and forgot to update here - just made this change